include/zephyr/xen/generic.h

```diff
@@ -32,4 +32,10 @@
 				__ATOMIC_SEQ_CST) ? new : old; \
 })
 
+#define NR_MAGIC_PAGES 4
```

Comment:
drop this commit and move changes to zephyr xen-lib
@firscity @oleksiimoisieiev